views/about.html

```diff
@@ -1,72 +1,53 @@
-            La volonté d'apprendre et d'échanger caractérise ces membres..
+            Les membres sont des passionnés, des experts, des curieux, des professionnels de tous horizons: 
+            éditeurs, sociétés industrielles ou commerciales, startups, indépendants, sociétés de service...
+            Ils sont réunis par la volonté d'apprendre et d'échanger.
         </p>
 
+        <h2>Rejoignez nous !</h2>
+        <h3>sur Slack</h3>
+        <p>Pour discuter avec les membres de nos communautés sur Slack, <b><a href="https://slack.lyontechhub.org/">inscrivez-vous</a></b> sur la page : <b><a href="https://slack.lyontechhub.org/" title="Inscription sur le Slack de Lyon Tech Hub">https://slack.lyontechhub.org</a></b>.</p>
+        <h3>en vrai</h3>
         <p>
-            Initié sous forme d'un calendrier Google et d'une liste de diffussion, nous avons souhaité augmenter sa
-            visibilité pour tous, afin de simplifier la découverte de nouveaux événements et de faciliter les
-            rencontres.
-        </p>
-    </div>
-
-    <div class="container">
-        <h2>Comment nous contacter ?</h2>
-
-        <p>
-            <ul>
-                <li>Sur <b>twitter <a title="@lyontechhub" href="https://twitter.com/LyonTechHub">@lyontechhub</a></b></li>
-                <li>Par <b>mail : <a title="contact@lyontechhub.org" href="mailto:contact@lyontechhub.org">contact@lyontechhub.org</a></b></li>
-            </ul>
-        </p>
-    </div>
-
-    <div class="container">
-        <h2>Vous représentez une communauté ou vous êtes aussi un passionné? <br /><a title="Inscription" href="https://groups.google.com/forum/?hl=fr#!forum/lyontechhub">Rejoignez-nous!</a></h2>
-
-        <p>
-            <b>Présenter-vous/présentez votre communauté</b> sur la liste de discussion, ou <b>mieux venez à des événements</b>, et
-            <b><a href="https://groups.google.com/forum/#!forum/lyontechhub">inscrivez-vous sur la liste</a></b> si vous souhaitez suivre les discussions.
+            N'hésitez pas à participer aux <b>évenements</b> organisés par les communautés de Lyon Tech Hub ! Rendez-vous sur la page <a href="/#!/calendrier">Calendrier</a>.
         </p>
 
+        <h3>Référencer une communauté</h3>
         <p>
-            Pour inscrire votre communauté, vous n'aurez qu'à faire une <b>pull request sur GitHub</b> pour mettre en ligne les
-            informations concernant votre communauté. Si vous ne savez pas faire une pull request sur GitHub, regarder
-            les
-            informations à fournir ci-dessous (prendre exemple sur les fichiers existants) et envoyez-les par mail à la
-            liste.
-            Demandez aussi sur la liste de diffusion <b>l'accès au calendrier Google</b> pour un ou plusieurs comptes Google.
-            <br />Voici comment faire la pull request:
+            Pour inscrire votre communauté, envoyez une <b>pull request sur GitHub</b> pour mettre en ligne les

```

Comment:
c'est un peu brutal les coupes dans le texte non ? (avec le bon compte...)

Reply:
Je trouve pas cela choquant, cela tiens sur l'écran au moins ^^

Reply:
Pour répondre, j'ai enlevé deux choses : 

1. "Présentez-vous"
Je pense que ce n'est pas nécessaire sur Slack comme l'app notifie déjà des arrivées sur le serveur. La division en channels permet de se regrouper selon ses centres d'interêts sans devoir broadcaster sa présence aux 500+ membres. 

2. comment s'ajouter au calendrier. C'est important mais je pense qu'il faut qu'on définisse un nouveau process qui ne passe pas les mails / la Mailing-List. En plus Google a remis à zéro les droits sur les calendriers partagés quand ils ont fait [leur refonte](https://www.blog.google/products/g-suite/time-refresh-introducing-new-look-and-features-google-calendar-web/) donc je ne sais même pas qui a les droits.
En pratique les gens arrive à trouver quelqu'un pour s'ajouter en demandant dans le chan #lth ou #general.
Je pense que ça serait pas mal d'écrire la procédure directement sur la page calendrier plutôt que sur le about qui est un peu fourre-tout (la même chose pour les groupes - [tout ce qui est à partir de la ligne 23](https://github.com/cappert/lyontechhub.github.io/blob/f7deef002e37aef046bff123477c37b95b0714ee/views/about.html#L23-L51)).